src/wp-admin/options-media.php

```diff
@@ -156,12 +167,18 @@
 <td colspan="2" class="td-full">
 <?php else : ?>
 <tr>
-<td class="td-full">
+<th scope="row"><?php _e( 'How do you want your uploads organized?' ); ?></th>
+<td class="td-full uploads_use_yearmonth_folders">
 <?php endif; ?>
-<label for="uploads_use_yearmonth_folders">
-<input name="uploads_use_yearmonth_folders" type="checkbox" id="uploads_use_yearmonth_folders" value="1"<?php checked( '1', get_option( 'uploads_use_yearmonth_folders' ) ); ?>>
-	<?php _e( 'Organize my uploads into month- and year-based folders' ); ?>
-</label>
+
+<input id="uploads_use_one_folder" name="uploads_use_yearmonth_folders" type="radio" value="0"<?php checked( empty( $storefolders ), true ); ?>>
+<label for="uploads_use_one_folder"><?php _e( 'Store all uploads in the same folder.' ); ?></label><br>
+
+<input id="uploads_use_year_folders" type="radio" name="uploads_use_yearmonth_folders" value="year"<?php checked( 'year', $storefolders ); ?>>
```

Comment:
The current options for this setting are '0' and '1', I think it would be more consistent if we used '2' for the new year option.